lib/agent0/agent0/hyperdrive/interactive/interactive_hyperdrive.py

```diff
@@ -76,6 +77,28 @@
 # pylint: disable=too-many-lines
 
 
+def running_interactive():
```

Comment:
move this into the script -- lets generalize it if we find ourselves using it too much first. Grugbrain recommends no premature abstraction. WET, then DRY.

Reply:
done in 916c248a7d9a9f3f45aef2079e75da332b01a5f1